src/main/resources/db/migration/V1__imrt_ddl.sql

```diff
@@ -66,3 +66,20 @@ CREATE TABLE stim_link (
   FOREIGN KEY (item_key_stim) REFERENCES item (key),
   PRIMARY KEY (item_key, item_key_stim)
 );
+
+/* Project Locking */
+CREATE TABLE project_lock (
+  project_id INT         NOT NULL,
+  lock_time  TIMESTAMPTZ NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_by VARCHAR     NOT NULL,
+  PRIMARY KEY (project_id)
+);
+
+/* flywayClean deletes all user privleges, so we will set them here.
+   Assumes that that imrt-ingest and imrt-search users have already been created in the database */
+GRANT SELECT, INSERT, UPDATE, DELETE, REFERENCES ON ALL TABLES in schema public to "imrt-ingest";
```

Comment:
Is this something that would be better in [beforeMigrate](https://flywaydb.org/documentation/callbacks.html)?

Reply:
I can't really tell from the documentation exactly when beforeMigrate will run - do you know how this is different from just putting it in the initial version? I was thinking if you did a clean, the initial version would then be the first thing that was applied. So this would go the first run, and then anytime there was a clean. AfterClean might also work, but I'm not sure that would go on the first run. I can play around with it for a bit if you like.

Reply:
This can be in v1.  I didn't realize when I reviewed that we're modifying an already existing migration.  So I thought you'd want this to run before any migration.